pulseaudio-ctl: add page (#18461)
tldr-pages/tldr: pages/linux/pulseaudio-ctl.md

# pulseaudio-ctl

> Control PulseAudio volume from the command line.
> More information: <https://github.com/graysky2/pulseaudio-ctl>.

- Increase volume by 5%:

`pulseaudio-ctl up`

- Increase volume by a specific amount:

> `pulseaudio-ctl up {{10}}`

Reviewer: ```suggestion
`pulseaudio-ctl up {{amount}}`
```

- Decrease volume by 5%:

`pulseaudio-ctl down`

- Decrease volume by a specific amount:

> `pulseaudio-ctl down {{10}}`

Reviewer: ```suggestion
`pulseaudio-ctl down {{amount}}`
```

- Set volume to a specific percentage:

> `pulseaudio-ctl set {{50}}`

Reviewer: ```suggestion
`pulseaudio-ctl set {{percentage}}`
```

> - Set volume to at most a specific percentage:

Reviewer 2: ```suggestion
- Set volume to a specific percentage if the current volume is higher than the provided value:
```

> `pulseaudio-ctl atmost {{100}}`

Reviewer: ```suggestion
`pulseaudio-ctl atmost {{percentage}}`
```
